test/acceptance/behave/components/ngsiv2/api_entry_point/retrieve_api_resource.feature

```diff
@@ -45,7 +45,7 @@ Feature: general operations in Context Broker using NGSI API v1 and v2
   Actions After the Feature:
   Setup: stop ContextBroker
 
-  @version
+  @version @skip
```

Comment:
This test is skipped because verifies the version installed, and then it is necessary to modify the version value in the configuration manually with each modification. I will append another test that verify the version structure (x.y.z[-next]) using Regular expression.